user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Tables/Table_component.md

```diff
@@ -4,229 +4,241 @@ uid: DashboardTable
+##### Column appearance presets
 
-- From DataMiner 10.3.8/10.4.0 onwards, a pop-up window will open where you can select whether the raw values or the display values from the table should be exported. Exporting the display values will result in a CSV file that contains all the values as they are seen in the table, formatted and with units. If you export the raw values, no formatting will be applied to them. The only exception are discrete values, for which the corresponding display values will always be exported. If no rows are selected in the table, the entire table will be exported; otherwise only the selected rows will be exported.
+To use one of the available presets to alter the column appearance, click the preview below the column name and select a preset option:
 
-The export file will be named “Query XXX” (XXX being the name of the query, as configured in the *Data* pane). The first line of the CSV file will contain the names of the columns. The subsequent lines will contain the data, each line being a row of the query result.
+| Preset | Description |
+|--|--|
+| ![Left](~/user-guide/images/Preset_Left.png) | Aligns text to the left of the cell. This is the default setting for columns containing values of type string. |
+| ![Center](~/user-guide/images/Preset_Center.png) | Centers the text within the cell. |
+| ![Right](~/user-guide/images/Preset_Right.png) | Aligns text to the right of the cell. This is the default setting for columns containing values of type double or datetime. |
+| ![Hyperlink](~/user-guide/images/Preset_Hyperlink.png) | Available in low-code apps only. Turns the text into a hyperlink that opens a webpage in a new tab. |
+| ![Icon](~/user-guide/images/Preset_Icon.png) | Displays a centered icon in the cell. By default, the info icon is used. |
+| ![Background](~/user-guide/images/Preset_Background.png) | Adds a background color to the cell. By default, a blue color (#1F68BF) is used. |
 
 > [!NOTE]
-> To only export specific columns, first apply a filter by dragging the columns onto the table component before you export the component.
-
-## Filtering and sorting the table
-
-From DataMiner 10.2.7/10.3.0 onwards, users can filter and sort the contents of a table component in a dashboard.
-
-> [!TIP]
-> If you have made changes to the way a table is displayed, and you want to quickly reset your changes and return to the initial table view, click the eye icon in the top-right corner of the component (available from DataMiner 10.2.11/10.3.0 onwards).
+> If you select the *Hyperlink* preset option, a [*Navigate to a URL* action](xref:LowCodeApps_event_config#navigating-to-a-url) is automatically configured. The default URL is `https://[Your DMA]/{DMA root}`, which you can edit using the [Template Editor](xref:Template_Editor).
```

Comment:
Default URL is no longer the one mentioned here. It's just interpretting the text as a URL (assuming it's something like 'http://www.skyline.be').